tests/CMakeLists.txt

```diff
@@ -791,10 +791,29 @@ foreach(prec IN LISTS TEST_PRECS)
                      --gtest_output=xml:gauge_force_test_${prec}.xml)
   endif()
 
+  if(QUDA_DIRAC_STAGGERED)
+    add_test(NAME unitarize_link_${prec}
+             COMMAND ${QUDA_CTEST_LAUNCH} $<TARGET_FILE:unitarize_link_test> ${MPIEXEC_POSTFLAGS}
+                     --dim 2 4 6 8 --prec ${prec}
+                     --gtest_output=xml:unitarize_link_test_${prec}.xml)
+  endif()
+
+  if(QUDA_FORCE_HISQ)
+    add_test(NAME hisq_paths_force_${prec}
+             COMMAND ${QUDA_CTEST_LAUNCH} $<TARGET_FILE:hisq_paths_force_test> ${MPIEXEC_POSTFLAGS}
+                     --dim 2 4 6 8 --prec ${prec}
```

Comment:
Can we reasonably test his force on 2 4 6 8 ? The 2 seems to be a bit short.

Reply:
The 2 is fine here.  Since the force algorithms all use extended fields, and we never use a gauge extension more than 2.